### ISSUE TITLE
Demo template management command unexpected args.

### ISSUE THREAD

**Issue by mathemancer** [centerofci/mathesar#2526]
## Description

After starting dev environment, the management command to setup the demo DB is broken. Trying to run:
```sh
# docker exec -it mathesar_service_dev python manage.py setup_demo_template_db
```
results in:
```
Traceback (most recent call last):
  File "/code/manage.py", line 22, in <module>
    main()
  File "/code/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 330, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 371, in execute
    output = self.handle(*args, **options)
  File "/code/demo/management/commands/setup_demo_template_db.py", line 15, in handle
    _setup_demo_template_db(*args, **options)
TypeError: _setup_demo_template_db() got an unexpected keyword argument 'verbosity'
```

## Expected behavior

You should be able to run the command listed above successfully in the `dev` environment.

## To Reproduce

Start the dev environment with a fresh docker state. Run the above command.

## Additional context

The problem is in line 15 of `demo/management/commands/setup_demo_template.py`.

**Comment by kgodey** [comment]
This is fixed in https://github.com/centerofci/mathesar/pull/2525, I think.

**Comment by mathemancer** [comment]
> This is fixed in #2525, I think.

You're correct; I linked that PR accordingly.
